test/com/googlecode/jmxtrans/model/output/CloudWatchWriterTests.java

```diff
@@ -0,0 +1,30 @@
+package com.googlecode.jmxtrans.model.output;
+
+import com.googlecode.jmxtrans.model.Query;
+import com.googlecode.jmxtrans.model.Server;
+import com.googlecode.jmxtrans.model.ValidationException;
+import org.junit.Test;
+
+/**
+ * Tests for {@link com.googlecode.jmxtrans.model.output.CloudWatchWriter}.
+ *
+ * @author Sascha Moellering <moelleri@amazon.de>
+ */
+public class CloudWatchWriterTests {
```

Comment:
In jmxtrans2, we have an agent with no external dependencies, but we do accept additional writers with additional dependencies. Those additional dependencies are packaged as separate jars, that you can use to enrich the agent. Have a look if you think that might be of use to you.

Note that jmxtrans2 is still work in progress. It should run fine, but it is likely that incompatible changes will still be introduced. In particular, configuration format will probably change in the near future.